internal/store/job.go

```diff
@@ -262,6 +262,24 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_job_status_duration_time",
+			Type: metric.Gauge,
+			Help: "DurationTime represents time when the job was duration.",
+			GenerateFunc: wrapJobFunc(func(j *v1batch.Job) *metric.Family{
+				ms := []*metric.Metric{}
+				if j.Status.CompletionTime != nil && j.Status.StartTime != nil{
+					ms = append(ms, &metric.Metric{
+
+						Value: j.Status.CompletionTime.Sub(j.Status.StartTime.Time).Seconds(),
```

Comment:
we typically don't do any computation in kube-state-metrics, why is using the metrics for each of these values not sufficient?

Reply:
Although Duration can be obtained with the `kubectl get jobs` command, since it is internally calculated, this method has been followed.
```
NAME                                              COMPLETIONS   DURATION   AGE
example-cronjob-1560267000                       0/1           19d        19d
example-cronjob-1561735800                       1/1           15s        2d15h
example-cronjob-1561822200                       1/1           22m        39h
example-cronjob-1561908600                       1/1           17m        15h
```
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/describe/versioned/describe.go#L2092-L2094

Reply:
not saying that's a wrong calculation, rather saying we can already do that with

```
kube_job_status_completion_time - kube_job_status_start_time
```